test/libsolidity/smtCheckerTests/types/array_branches_3d.sol

```diff
@@ -15,12 +15,6 @@ contract C
 }
 // ====
 // SMTEngine: all
-// SMTIgnoreOS: macos
+// SMTTargets: assert
```

Comment:
The out of bounds checks seem to be difficult for Z3, but there is no point to test them, they are safe.
The point of this test is to test the assertion.